biogrid/apps/biogrid-mvp/src/app/pages/info-page/info-page.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import './info-page.css';
+import Visual from './EnergyFlowVisual.png';
+
+export const InfoPage = () => {
+
+
+  return (
+    <div className="info-page">
+      <h1>The Logic</h1>
+
+      <img src={Visual} id="EnergyFlow"></img>
+
+      <p>Firstly, the grid items <b>(batteries(b), solar panels(s), and buildings(b))</b> will be randomly placed in the town. 
+      The grid items include the <b>grid(g)</b> itself as well. In case of small batteries, we will refer to them as <b>s_b</b>, and large batteries as <b>l_b</b>.
+      </p>
+
+      <p>Then they will be connected by a <b>directed graph G</b>, which connects the grid items from the grid to these items except the solar panels i.e <b>G(g, b), G(g, b)</b>. 
```

Comment:
"then they will be connected" -> who is they? do you mean all grid items?

Reply:
Yep sorry about that